lib/ansible/modules/cloud/amazon/elb_application_lb_facts.py

```diff
@@ -0,0 +1,260 @@
+            returned: when status is present
+            type: string
+            sample: 60
+        ip_address_type:
+            description:  The type of IP addresses used by the subnets for the load balancer.
+            returned: when status is present
+            type: string
+            sample: ipv4
+        load_balancer_arn:
+            description: The Amazon Resource Name (ARN) of the load balancer.
+            returned: when status is present
+            type: string
+            sample: arn:aws:elasticloadbalancing:ap-southeast-2:0123456789:loadbalancer/app/my-elb/001122334455
+        load_balancer_name:
+            description: The name of the load balancer.
+            returned: when status is present
+            type: string
+            sample: my-elb
+        scheme:
+            description: Internet-facing or internal load balancer.
+            returned: when status is present
+            type: string
+            sample: internal
+        security_groups:
+            description: The IDs of the security groups for the load balancer.
+            returned: when status is present
+            type: list
+            sample: ['sg-0011223344']
+        state:
+            description: The state of the load balancer.
+            returned: when status is present
+            type: dict
+            sample: "{'code': 'active'}"
+        tags:
+            description: The tags attached to the load balancer.
+            returned: when status is present
+            type: dict
+            sample: "{
+                'Tag': 'Example'
+            }"
+        type:
+            description: The type of load balancer.
+            returned: when status is present
+            type: string
+            sample: application
+        vpc_id:
+            description: The ID of the VPC for the load balancer.
+            returned: when status is present
+            type: string
+            sample: vpc-0011223344
+'''
+
+import traceback
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import (boto3_conn, boto3_tag_list_to_ansible_dict, camel_dict_to_snake_dict,
+                                      ec2_argument_spec, get_aws_connection_info)
+
+try:
+    import boto3
+    from botocore.exceptions import ClientError, NoCredentialsError
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+
+def get_load_balancer_attributes(connection, module, load_balancer_arn):
+
+    try:
+        load_balancer_attributes = boto3_tag_list_to_ansible_dict(connection.describe_load_balancer_attributes(LoadBalancerArn=load_balancer_arn)['Attributes'])
+    except ClientError as e:
+        module.fail_json(msg=e.message, exception=traceback.format_exc(), **camel_dict_to_snake_dict(e.response))
+
+    # Replace '.' with '_' in attribute key names to make it more Ansibley
+    for k, v in load_balancer_attributes.items():
+        load_balancer_attributes[k.replace('.', '_')] = v
```

Comment:
Can you document this in the examples? Seems like the sort of thing that will trip folks up.